_includes/3-provisioning.md

```diff
@@ -185,13 +185,15 @@ This list is not exhaustive and is detailed in the <span class="h5">Request body
 The following HTTP request is sent from the provider to Ozwillo. 
 
 <pre>
-POST /apps/pending-instance/{instance_id} HTTP/1.1
+POST {instance_registration_path} HTTP/1.1
 Host: kernel.ozwillo-preprod.eu
```

Comment:
We should use `{instance_registration_host}` here, as we did for `{instantiation_uri}` and `{destruction_uri}`.